README.md

```diff
@@ -100,7 +111,109 @@ Default: `nil`
 
 ### ChatOps options
 
-API exists, but is not implemented yet
+#### `enabled`
+
+Type: boolean
+Description: Enable/Disable chatops support
+Default: false
+
+#### `service`
+
+Type: string
+Description: Which service to use. Supported options: [`slack`, `rocketchat`]
+Default: ""
+
+#### `channel`
+
+Type: string
+Description: ChatOps communication channel to post to.
+Default: ""
+
+#### `user`
+
+Type: string
+Description: ChatOps user to post as
+Default: ""
+
+#### `auth_token`
+
+Type: string
+Description: The authentication token needed to post as the ChatOps user in the chosen, supported ChatOps service
+Default: ""
+
+#### `server_uri`
+
+Type: string
+Description: The ChatOps service API URI to send the message to.
+Default: ""
+
+### Orchestration
+
+#### `enabled`
+
+Type: boolean
+Description: Enable/Disable orchestration support
+Default: false
+
+#### `type`
+
+Type: string
+Description: Which orchestration tool to use. Currently only supports `bolt`.
+Default: ""
+
+#### `user`
+
+Type: string
+Description: User to authenticate to the target as.
+Default: ""
+
+#### `password`
+
+Type: string
+Description: Password for authentication use.
+Default: ""
+
+#### `bolt`
+
+Type: hash
+Description: Hash of Puppet Bolt specific settings.
+Default: `nil`
+
+##### `transport`
+
+Type: string
+Description: What kind of bolt network transport protocol to use. Supported types: [`local`, `ssh`] with more coming soon.
+Default: `nil`
+
+##### `targets`
+
+Type: array
+Description: A list of target nodes that Bolt will attempt to run `r10k` on.
+Default: []
+
+##### `concurrency`
+
+Type: integer
+Description: Maximum number of simultaneous connections.
+Default: 100
+
+##### `run_as`
+
+Type: string
+Description: User to run the `r10k` command as.
+Default: ""
+
+##### `sudo_password`
+
+Type: string
+Description: The password to use when using `sudo` to run `r10k` as the `run_as` user.
+Default: ""
+
+##### `HostKeyCheck`
+
+Type: boolean
+Description: Enable/Disable SSH host key checking
+Default: false
```

Comment:
from a security perspective I recommend setting this to true by default

Reply:
Until I can find a way to have those host keys pre-populated, then it has to default to false since users cannot provide user input to the running webserver